readme.md

```diff
@@ -101,3 +101,176 @@ In the console...
+    ```ruby
+    get '/' do
+      "Hello World"
+    end
+    ```
+  Now head back to localhost:4567 and check it out!
+
+  - So maybe not super exciting but moving in the right direction
+
+  - Replace `"Hello World"` with HTML rendered from an ERB template, home. Our General Routes section of app.rb now reads:
+    ```ruby
+    get '/' do
+      erb :home
+    end
+    ```
+
+- [erb](http://www.sinatrarb.com/intro.html#Views%20/%20Templates) is a method provided by Sinatra which uses Ruby's [ERB (Embedded RuBy)](https://en.wikipedia.org/wiki/ERuby) templating system to take a template string (text with ruby written in it) and produce an output string after evaluating the embedded ruby.
+
+- If we tried to go to localhost:4567 now, we'd get an error saying __No such file or directory @ rb_sysopen - /path/to/tunr_sinatra/views/home.erb__
+
+    - When we give the `erb` method an argument of `:home`, the method looks in the __views/__ directory of the project for a file named home.erb and takes the contents of that file as its template string.
+
+    - Make that __views/__ directory and add a file home.erb.
+
+    - In home.erb, first add some HTML (no need to worry about embedding ruby just yet) and check it out at localhost:4567
+
+- The erb method will also check for a layout.erb file where you define the part of the template shared by all the pages in our app (e.g. head/body elements, header/footer, navigation bar, etc). In layout.erb need to include a line of embedded ruby with the keyword `<%= yield %>`. This tells the method to render the named template (`:home`) at this point in the layout.erb template. In the below steps, regularly check out localhost:4567 and consider the effects of your code changes
+
+    - Make a layout.erb file in views/
+
+    - As always, start simple, maybe just a heading. Check it out in the browser
+
+    - Then expand adding head and body elements, layout.erb should produce well formed HTML and since there is no embedded ruby in it yet, it should be well formed HTML
+
+    - Create a __public/__ directory in the root of your project. By default, Sinatra looks for static assets in a directory named __public/__ in the root directory of the project. In the __public/__ create a stylesheet.
+
+    - Back in layout.erb add a link to your stylesheet in the head. In the body add a nav element containing links to '/artists' and '/songs'. Finally below the nav element, add an embedded ruby expression calling `yield`.
+
+    - Take a moment using your browser's dev tools to look at how the HTML rendered in from home.erb nests into the HTML rendered from layout.erb where we included `<%= yield %>`.
+
+    - Spend a few minutes making this page look nice/pleasant -- as you keep working on the app, you're going to see it a whole bunch
+
+### Part 3.2: Artist Controller and Views
+
+1. Make a __controllers/__ directory in __tunr_sinatra/__ and create a file __artists.rb__ which will hold the routes for your artists
+
```

Comment:
is controllers a new term? maybe give a brief definition: it's the c in MVC. We'll organize our routes and their actions by controllers